packages/workbox-webpack-plugin/src/lib/warn-about-config.js

```diff
@@ -0,0 +1,54 @@
+/*
+  Copyright 2017 Google Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+module.exports = (config) => {
+  const moreInfoUrl = 'https://goo.gl/EQ4Rhm';
+
+  const globOptions = [
+    'globDirectory',
+    'globFollow',
+    'globIgnores',
+    'globPatterns',
+    'globStrict',
+  ];
+
+  const usedGlobOptions = globOptions.filter((option) => option in config);
+  if (usedGlobOptions.length > 0) {
+    return `You're using the following Workbox configuration options: ` +
```

Comment:
Maybe `You're using **one** of the following workbox configuration`

Reply:
The error message only includes the options that are actually being used, so I think it's accurate. The only nit is whether `options` should be singular when there's one option... so I can change that.